roles/rook-ceph/defaults/main.yaml

```diff
@@ -1,2 +1,16 @@
 rook_ceph_namespace: rook-ceph
 rook_ceph_version: "1.16.6"
+
+ceph_cluster:
```

Comment:
this gets generated by `./roles/rook-ceph/generate-rook-ceph-cluster-config.sh`
I am not sure we want it in the default section.